www/src/components/searchbar-body.js

```diff
@@ -0,0 +1,225 @@
+import React, { Component } from 'react'
+import { InstantSearch, Hits, SearchBox, Stats, RefinementList, PoweredBy } from 'react-instantsearch/dom'
+import distanceInWords from 'date-fns/distance_in_words'
+import presets, { colors } from "../utils/presets"
+import Link from 'gatsby-link'
+import DownloadsBlack from '../assets/down-arrow.svg'
+import DownloadsWhite from '../assets/down-arrow-white.svg'
+import {debounce} from 'lodash'
+
+import typography, { rhythm } from "../utils/typography"
+
+// This is for the urlSync
+const updateAfter = 700
+//
+
+const Search = ({searchState}) => {
+  const emptySearchBox = searchState.length > 0 ? false : true
+
+  return (
+    <div className="container">
+      <div css={{
+        display: `flex`,
+        justifyContent: `center`,
+      }}>
+
+      <SearchBox
+        translations={{placeholder: 'Search Gatsby Library'}}
+      />
+
+      </div>
+
+      <div css={{
+        display: `none`
+      }}>
+        <RefinementList attributeName="keywords"
+          defaultRefinement={["gatsby-component", "gatsby-plugin"]}
+        />
+      </div>
+
+
+     <div css={{
+       opacity: emptySearchBox ? 0 : 1,
+       height: rhythm(1.5),
+       paddingTop: rhythm(.25),
+       paddingBottom: rhythm(.25),
+     }}>
+       <Stats
+         translations={{
+           stats: function(n, ms){
+             return `${n} results`
+           }
+         }}
+       />
+     </div>
+
+     <div css={{
+       borderTop: `2px solid #F5F3F7`,
+       borderBottom: `2px solid #F5F3F7`,
+       borderLeft: `2px solid #F5F3F7`,
+     }}>
+
+        <div css={{
+          backgroundColor: `white`,
+          height: `calc(100vh - 225px)`,
+          border: `2 px solid red`,
+          overflowY: `scroll`,
+          WebkitOverflowScrolling: `touch`,
+          "::-webkit-scrollbar": {
+            width: `6px`,
+            height: `6px`,
+          },
+          "::-webkit-scrollbar-thumb": {
+            background: colors.ui.bright,
+          },
+          "::-webkit-scrollbar-track": {
+            background: colors.ui.light,
+          },
+        }}>
+          <Hits hitComponent={Result} />
+        </div>
+
+    </div>
+
+      <div>
+        <h3 css={{
+          fontSize: rhythm(.55),
+          textAlign: `center`,
+          margin: rhythm(.75),
+        }}>Search by <a href={`https://www.algolia.com/`} style={{color: `#744C9E`, border: `none`, boxShadow: `none`}}>Algolia</a></h3>
```

Comment:
However you decided, but it's also possible to link to the post to how it's built on the forum: https://discourse.algolia.com/t/2016-algolia-community-gift-yarn-package-search/319/1

Reply:
I for one am eternally grateful for all the help so would defer to you and Algolia. Would the forum post be your preference @Haroenv?

Reply:
Actually the homepage works best, but you can add `/?utm_campaign=poweredby&utm_content=gatsby`, so we can set up a dedicated page for it